examples/kotlin-diktat/sarif-actual-expected/save.toml

```diff
@@ -0,0 +1,21 @@
+[general]
+    tags = ["warn", "sarif"]
+    description = "Test warnings discovered by diKTat, reading expected warnings from SARIF file"
+    suiteName = "Only Warnings: with SARIF format"
+
+[warn]
+    # regular expression to detect tests
+    testNameRegex = ".*Test.kt"
+    expectedWarningsFormat = "SARIF"
+    expectedWarningsFileName = "save-warnings-expected.sarif"
+    actualWarningsFormat = "SARIF"
+    actualWarningsFileName = "/Users/vladislav/IdeaProjects/save/examples/kotlin-diktat/sarif-actual-expected/save-warnings-actual.sarif"
```

Comment:
please support relative path from the test root path (the directory where the binary is stored and run)

Reply:
Fixed by using `calculatePathToSarifFile()` method in warn plugin